cmd/Makefile.am

```diff
@@ -338,6 +338,7 @@ snap_confine_unit_tests_SOURCES = \
 	snap-confine/snap-confine-args-test.c \
 	snap-confine/snap-confine-invocation.c \
```

Comment:
This is incorrect, when you are adding -test.c you must remove the non test .c and .h file since the test file replaces them entirely.

Reply:
Fixed